packages/edit-site/src/components/header/style.scss

```diff
@@ -9,11 +9,16 @@
 .edit-site-header__toolbar {
 	display: flex;
 	flex-grow: 1;
-	padding-left: $grid-unit-10;
+	padding-left: $grid-unit-30;
 	align-items: center;
 
-	& > * {
+	.edit-site-header-toolbar__inserter-toggle {
```

Comment:
Why doesn't `edit-post-header__toolbar` have similar styles?

Reply:
It does, but it's too specific and expects to be in a `post-editor` container. :/

Reply:
I only see it when it has `.has-icon`.

Reply:
@epiqueras What do you mean. I don't fully understand what you're asking or proposing here.

Reply:
`edit-post-header__toolbar` only has these styles with `.has-icon`. Shouldn't it be the same here?

Reply:
It looks like the `.has-icon` selector is unnecessary. This element always has an icon. I can't think of a good reason to add it other than maybe it was an attempt to increase specificity to protect against theme style conflicts. Feel free to add it.

Reply:
Can we keep it consistent between both places? I.e., remove it from the post editor.

Reply:
Sure. @epiqueras that should probably be a separate PR, correct?

Reply:
Sure